### PR TITLE
Add translations and layer updates for RBD deployment

### PR DESCRIPTION
**Pull request by wadhwamatic**
This PR makes a few changes to the configuration of the Regional Bureau Dakar (RBD) deployment:
1. Change boundary files to utilize boundaries hosted on HDX: https://data.humdata.org/dataset/west-and-central-africa-administrative-boundaries-levels
2. Temporarily remove vulnerability layers
3. Utilize menu grouping for rainfall indicators
4. Add first version of French translations 

Surge deployment: https://demo-prism-rbd-v4.surge.sh/

### CODE REVIEW

**Comment by ericboucher** [comment]
@wadhwamatic can you add a deploy link? :)

File: src/config/rbd/translation.json [hunk 1]
```diff
@@ -19,5 +19,88 @@
...
+  "Total number of heavy rain days (rainfall > 95th percentile) within last 30 days of dekad": "Nombre total de jours de fortes pluies (pluies > 95e centile) au cours des 30 derniers jours de la décade",
+  "Longest consecutive number of heavy rain days (rainfall > 75th percentile) within last 30 days of dekad": "Plus long nombre consécutif de jours de fortes pluies (pluies > 75e centile) au cours des 30 derniers jours de la décade",
+  "Normalized Difference Vegetation Index (NDVI) derived from MODIS TERRA/AQUA": "Indice de végétation par différence normalisée (NDVI) dérivé de MODIS TERRA/AQUA",
+  "Normalized Difference Vegetation Index (NDVI) anomaly when compared to the long-term average": "Anomalie de l'indice de végétation par différence normalisée (NDVI) par rapport à la moyenne à long terme",
+  "Daytime land surface temperature": "Température diurne de la surface terrestre",
+  "Daytime land surface temperature anomaly when compared to long-term average": "Anomalie de la température diurne de la surface terrestre par rapport à la moyenne à long terme"
```
Comment (ericboucher) [inline src/config/rbd/translation.json:105]: `de la surface terrestre` can be simplified -> `au sol`

**Comment by wadhwamatic** [comment]
> @wadhwamatic can you add a deploy link? :)

added! https://demo-prism-rbd-v4.surge.sh/

File: src/config/rbd/translation.json (hunk 1 above)
Comment (wadhwamatic) [reply src/config/rbd/translation.json:105]: updating, thanks!

**Pull request merged**